polar-core/src/polar.lalrpop

```diff
@@ -264,7 +256,7 @@ Exp10<T>: Term = {
 }
 
 CallTerm: Value = {
-    <SimpleCall>,
```

Comment:
Looks like there might be some additional cleanup of `SimpleCall` stuff we could do in this file. I'm not sure why `Value` still accepts a `SimpleCall` on 491:
```
pub Value: Value = {
    <Number>,
    <PolarString>,
    <Boolean>,
    <Variable>,
    <DictionaryTerm>,
    <BuiltinOperation>,
    <RewrittenOperation>,
    <SimpleCall>,
    <List<"Term">>,
};
```
or why we couldn't just collapse the declaration of `SimpleCall` into `Call` now that we don't need to refer to `SimpleCall` separately

Reply:
oh wait is the `SimpleCall` stuff still necessary since we parse Polar predicates (rule heads) the same as method/constructor calls?

Reply:
but wait that wouldn't make sense either since (I think) the `param: Specializer` fields in a Polar predicate require different semantics than method calls, which require all kwargs to be at the end of the arglist.

Reply:
We can kill `SimpleCall` and fold its cases into `Call`. Thanks for catching this.

Reply:
Word of caution: we do support prolog-style `person("sam") = person(name)` style unification, which is possibly what this syntax is referring to.

Reply:
`SimpleCall` is used for Polar predicates... I don't think it can be folded into `Call`

Reply:
We need it to be separate because of specializers, as Gabe pointed out.

Reply:
@plotnick and I talked about this -- we use `ParameterList` to capture the `param: Specializer` case in rule heads. We folded `SimpleCall` into `Call` and everything seems kosher.